packages/components/src/tooltip/index.tsx

```diff
@@ -128,7 +128,12 @@ function UnforwardedTooltip(
 			{ isOnlyChild && ( text || shortcut ) && (
 				<Ariakit.Tooltip
 					{ ...restProps }
-					className={ clsx( 'components-tooltip', className ) }
+					className={ clsx(
+						'components-tooltip',
+						{ 'move-up': text?.toLowerCase().includes( 'up' ) },
+						{ 'move-down': text?.toLowerCase().includes( 'down' ) },
```

Comment:
Separately, I don't think it makes sense to use the text to infer particular behavior. What if we change or remove those text labels? Sounds fragile. Instead, we should try to do that from any other logical props, like `BlockMoverButton`'s `direction` for example.